### ISSUE TITLE
Check that a `pp.Domain` is simply connected

### ISSUE THREAD

**Issue by jhabriel** [pmgbergen/porepy#902]
Checks for whether a domain is simply-connected should be added to the `pp.Domain` class. This should be straightforward for 2d, but it is slightly more involved for 3d.

Currently, this is taken as an assumption. However, no proper check is in place.

**Comment by keileg** [comment]
I agree we should ideally fix this, but is it really an issue which should be prioritized over other problems?

**Comment by jhabriel** [comment]
I would say it has a low priority, yes. I will add the the label.
